core/Metrics/Formatter.php

```diff
@@ -276,6 +278,79 @@ private function makeRegexToMatchMetrics($metricsToFormat)
         return '/^' . implode('|', $metricsRegexParts) . '$/';
     }
 
+    /**
+     * @todo For matomo 4, should really use the Metric class to house this kind of logic
```

Comment:
For Matomo 4?

Reply:
This only moves a comment that was there before 🙈